src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -12,6 +16,9 @@
   private static final String MOBILE_IN_CONTENT = "MOBILE_IN_CONTENT";
   private static final String MOBILE_PREFOOTER = "MOBILE_PREFOOTER";
   private static final String SRC = "mobile";
+  private static final String PORTABLE_INFOBOX = ".portable-infobox";
+  private static final String ARTICLE_HEADER = ".wiki-page-header";
+  private static final String MOBILE_TOP_LEADERBOARD_ID = "#MOBILE_TOP_LEADERBOARD";
```

Comment:
Could you use [AdsContent.java](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java)?

E.g:  
`AdsContent.MOBILETOP_LB` for getting slot name   
or  
`AdsContent.getSlotSelector(AdsContent.MOBILETOP_LB)` for getting slot selector